void init_exception(void);
